content/en/integrations/guide/data-center-migration.md

```diff
@@ -0,0 +1,158 @@
+
+{{% datadog-sync-resources-dependencies %}}
+
+At this point, you have a destination organization in a new data center which possesses a list of Datadog resources migrated from your old Datadog account. These resources are empty because they do not contain any monitoring data.
+
+### Limitations
+
+The `datadog-sync-cli` tool does not migrate intake data such as ingested logs or metrics, and does not migrate API keys or integrations.
+
+The following Datadog resources are not covered by the `datadog-sync-cli` tool:
+
+Log Management
+: Saved views, log facets, log JSON pre-processing attributes, and any increased log monitor quota (the default number of log monitors allowed is 1000). </br></br> To migrate existing logs to a new Datadog account, use [Log Archiving][2] and [Log Rehydration™][3].
+
+### Migrate integrations and real-time data
+
+Integrations
+: Migrating to another data center requires you to re-add your integration. Use a CI mechanism to mass update agents (by using the `logs_pipelines` resource).
+
+APM
+: Setting a second primary tag in APM requires you to update existing dashboards and monitor queries.
+
+Next, configure the sources that send data to Datadog (the Datadog Agent, API, and more) to point to the new data center. To maintain historical metrics and prevent data loss, you have two options:
+
+- Migrate to a data center and access historical metrics using the source organization. For example, if you are moving from `US1` to `US5`, you can keep your `US1` organization to access historical metrics. The data remains in the source organization until the metric retention period ends.
+- [Dual ship][4] metrics to two data centers until the new Datadog organization possesses the required volume of historical data. 
+
+### Use Dual Shipping
+
+The following products can use Dual Shipping:
+
+| Product |
+|---|
+| Infrastructure (Metrics, Events, Processes, Containers) |
+| Log Management |
+| APM |
+| Continuous Profiling |
+| Database Monitoring |
+| CI Visibility |
+| Network Performance Monitoring |
+| Network Device Monitoring |
+| Cloud Security Posture Management |
+| Cloud Workload Security |
+| Cloud SIEM |
+
+<div class="alert alert-warning">By sending twice the amount of data during the migration, you are billed as if you are maintaining two live Datadog instances.</div>
+
+The following products cannot use Dual Shipping:
+
+| Product |
+|---|
+| Synthetic Monitoring |
+| Continuous Testing |
+| Mobile Application Testing |
+| Incident Management |
+| Real User Monitoring |
+
+### Turn off Dual Shipping
+
+To avoid additional charges in the month associated with products that are billed for [Dual Shipping][4], Datadog recommends disabling hosts or integrations such as AWS and GCP once there is enough data in the destination organization before the next billing cycle starts on the first day of the calendar month at 12:00 am UTC.
```

Comment:
```suggestion
To avoid additional charges in the month associated with products that are billed for [Dual Shipping][4], Datadog recommends disabling hosts or integrations such as AWS and GCP after there is enough data in the destination organization before the next billing cycle starts on the first day of the calendar month at 12:00 am UTC.
```